api/src/test/java/org/openmrs/module/pihcore/reporting/BaseReportTest.java

```diff
@@ -67,7 +67,7 @@
     protected MetadataDeployService deployService;
 
     @Autowired
-    protected HaitiAddressBundle haitiAddressBundle;
+    protected TestAddressBundle testAddressBundle;
```

Comment:
Eventually we will want to set something up where we can initialize a test context using Iniz files, but here I just copied over the HaitiAddressBundle from Haiti Core, trimming the CSV to 5 lines, and renamed it "Test Address Bundle"